.github/workflows/ci.yml

```diff
@@ -406,6 +406,25 @@ jobs:
           password: ${{ secrets.PYPI_TOKEN }}
 
 
+  push_to_registry:
+    name: Push Docker image to Docker Hub
+    runs-on: ubuntu-latest
+    needs: [normal_test, large_test, mujoco_test, mujoco_test_long]
+    if: startsWith(github.ref, 'refs/tags/v')
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Push to Docker Hub
+        uses: docker/build-push-action@v1
+        with:
+          username: ${{ secrets.DOCKER_HUB_USERNAME }}
+          password: ${{ secrets.DOCKER_HUB_TOKEN }}
+          repository: rlworkgroup/garage
+          tag_with_ref: true
```

Comment:
would this tag with `v2020.06.1` or `refs/tags/v2020.06.1`? We want the former.

Reply:
https://github.com/docker/build-push-action/tree/releases/v1#tag_with_ref 
I believe it will tag correctly.